app/src/main/java/br/ufg/inf/pes/healthhelp/model/Profissional.java

```diff
@@ -14,6 +14,8 @@
     private Date ultimoAcesso;
     private String profissao;
     private StatusProfissional statusProfissional;
+    private Usuario usuario;
```

Comment:
Um profissional "é" um usuário. Como um usuário pode desempenhar diferentes papéis (exercer diferentes ocupações), então Usuario tem uma lista de Profissional. Quando navegamos no sentido Usuario -> Profissional, tudo acontecer corretamente. Entretanto, nesse branch eu notei que existe a situação em que se tem um Profissional mas você quer dados dele como usuário. Pra isso, eram duas as opções, buscar o Usuario no banco de dados para cada profissional (eles estão numa lista em que cada profissional é requisitado alguma vez quando navega-se pela lista) ou buscar o usuário juntamente com o profissional no momento da busca e permitir a navegação Profissional -> Usuario. Essa última opção me pareceu mais eficiente, menos complexa e mais fácil de coerente. Por isso implementei dessa forma.